theano/tensor/signal/tests/test_pool.py

```diff
@@ -35,8 +35,8 @@ def numpy_max_pool_2d(input, ds, ignore_border=False, mode='max'):
             if input.shape[-1] % ds[1]:
                 yi += 1
         out_shp = list(input.shape[:-2])
-        out_shp.append(input.shape[-2] / ds[0] + xi)
-        out_shp.append(input.shape[-1] / ds[1] + yi)
+        out_shp.append(int(input.shape[-2] / ds[0] + xi))
```

Comment:
It would be cleaner to explicitly call an integer division (`input.shape[-2] // ds[0]`) rather than do a float division and then cast.